ambassador/entrypoint.sh

```diff
@@ -177,11 +177,17 @@ ambex "${ENVOY_DIR}" &
 AMBEX_PID="$!"
 pids="${pids:+${pids} }${AMBEX_PID}:ambex"
 
+# Option to customize diagd binding
+if [ -z "$DIAGD_SERVERBIND" ]; then
+    DIAGD_SERVERBIND=0.0.0.0
+fi
+
 # We can't start Envoy until the initial config happens, which means that diagd has to start it.
 
 echo "AMBASSADOR: starting diagd"
 
 diagd "${SNAPSHOT_DIR}" "${ENVOY_BOOTSTRAP_FILE}" "${ENVOY_CONFIG_FILE}" $DIAGD_DEBUG $DIAGD_CONFIGDIR \
+      --host "DIAGD_SERVERBIND" \
```

Comment:
This will look for a host named `DIAGD_SERVERBIND`, which probably doesn't exist. :smile: Thinking you're missing a `$`...

Reply:
Ha :-O
Yes, definitely, sorry about that. Fixed.